File: Faltantes o por revisar/Generadores de casos de prueba/TestCasesGeneratingRandomArrays.cpp
#include <cstdlib>
#include <bits/stdc++.h> 
using namespace std;

int RUN = 5;
int lowerBound = 0;
int upperBound = 1000;
int minSize = 10;
int maxSize = 20;

void randomArray() {
    for (int i = 0; i < RUN; i++) {
        int size = maxSize - (rand() % minSize);
        int array[size];
        cout << size << endl;
        for (int j = 0; j < size; j++) {
            int a = upperBound - (rand() % upperBound);
            cout << a << " ";
        }
        cout << endl;
    }
}

int main() {
    randomArray();
}

File: Faltantes o por revisar/Generadores de casos de prueba/TestCasesGeneratingRandomArrays.py
import random as r
RUN = 5;
lowerBound = 0;
upperBound = 1000;
minSize = 10;
maxSize = 20;
def randomArray():
    for i in range (RUN):
        size = r.randint(minSize,maxSize)
        array=[0 for j in range(size)]
        print(size)
        str1=""
        for j in range (size):
            a = r.randint(lowerBound,upperBound)
            str1+=str(a)+" " 
        print(str1)
randomArray()        
        

    

